csharp/src/Apache.Arrow.Flight/Apache.Arrow.Flight.csproj

```
@@ -1,13 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```

We don't have any `#if`s in this project, so there is no reason to build it twice.

---

But then you'll depend on the `AsyncInterfaces` nuget even though they are not required for .NET Standard 2.1, right? Could be that I am misunderstanding the package resolution :)

---

Yeah, you're right. In reality that isn't a huge deal since the `Microsoft.Bcl.AsyncInterfaces` assembly on .NET Standard 2.1+ just contains contains TypeForwardTo attributes (it's about 16 KB). But I guess it would be one less dependency to download and deploy in .NET Core apps.